repo/testdata/config

```diff
@@ -49,11 +49,15 @@
     "PrivKey": "testKey"
   },
   "Ipns": {
+    "QuerySize": 0,
```

Comment:
That test data needs to be a properly formatted object for the test to pass. So it needs to have the QuerySize in there.